packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4469,7 +4479,19 @@ class EditableTextState extends State<EditableText>
       }
 
       spellCheckResults = SpellCheckResults(text, suggestions);
-      renderEditable.text = buildTextSpan();
+      final double? lineHeightScaleFactor = mounted
```

Comment:
why's the mounted check needed? Generally the state should unregister itself from these services in dispose?

Reply:
There were no mounted checks before.

Reply:
Because this logic is happening in an async method, the lint complained we were not checking mounted.